OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -2666,39 +2711,45 @@ def __init__(self) -> None:
     def getParameterNode(self):
         return OpenLIFUTransducerTrackerParameterNode(super().getParameterNode())
 
-    def pull_photocollection_from_android(self, reference_number: str) -> List[str]:
+    def pull_photocollection_from_android(self, reference_number: str) -> tuple[str, List[str]]:
```

Comment:
Since it can pull either a photoscan or a photocollection, this function should be renamed. Maybe something like `pull_photo_data_from_android`